sdk/eventhub/tests.yml

```diff
@@ -1,8 +1,41 @@
 trigger: none
 
 extends:
-  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests-multiple.yml
   parameters:
-    MaxParallel: 6
-    ServiceDirectory: eventhub
-    TimeoutInMinutes: 120
\ No newline at end of file
+    jobs:
+      # Azure Cloud
+      - ServiceDirectory: eventhub
+        MaxParallel: 6
+        TimeoutInMinutes: 120
+        ScenarioName: AzureCloud
+        SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+        EnvVars:
+          AZURE_AUTHORITY_HOST: https://login.microsoftonline.com
```

Comment:
I think setting these variables should be integrated into New-TestResources script. cc @danieljurek

Reply:
I've opened this to track: https://github.com/Azure/azure-sdk-tools/issues/757 

I think we might be able to make use of the $Environment parameter to infer this except in cases where the authority host is not well known (e.g. Azure Stack)